src/app/webshop/catalogue/catalogue.component.html

```diff
@@ -0,0 +1,48 @@
+<app-front-basic-layout>
+  <ng-container>
+    <p>catalogue works!</p>
+    <mat-grid-list cols="2" rowHeight="2:1">
+      <mat-grid-tile>
+        <mat-card>
+          <mat-card-content>
+            <mat-form-field>
+              <mat-label>Søg</mat-label>
```

Comment:
This should not be in a grid. Make a new component calles search component and make it standard + reusable. Perhaps an input for a list of filters and an output for search.